package.json

```diff
@@ -68,7 +69,7 @@
     "eslint-plugin-react-hooks": "^4.2.0",
     "eslint-watch": "^7.0.0",
     "husky": "^6.0.0",
-    "node-sass": "^5.0.0",
+    "node-sass": "^6.0.1",
```

Comment:
We'd better decouple dependencies upgrade into a separated PR.

Reply:
i can do this first, but this is a necessity in order to make the view run iirc.